munimap/static/js/geoeditor/geoeditor-popup-controller.js

```diff
@@ -1,12 +1,13 @@
 angular.module('munimapGeoeditor')
     .controller('geoeditorPopupController', ['$scope', '$rootScope', 'GeoeditorValidationService', 'munimapConfig',
         function ($scope, $rootScope, GeoeditorValidationService, munimapConfig) {
-            $scope.defaultPopupTabContent = 'geoeditor';
-            $scope.showGeoeditorTab = true;
+            $scope.defaultPopupTabContent = undefined; // default set to undefined, case there are no fields to show
+            $scope.showGeoeditorTab = false; // default set to false, case there are no fields to show
```

Comment:
where do we set these values then?

Reply:
Those are being set later in the file through some conditions. for instance:
```